app/assets/stylesheets/forever_style_guide/modules/_all.scss

```diff
@@ -13,11 +13,7 @@
 @import "input_groups";
 @import "list";
 @import "message_box";
-@import "nav";
-@import "nav-account_dropdown";
-@import "nav-cart";
-@import "nav-fixed";
-@import "nav-mobile-menu";
+@import "pnav";
```

Comment:
I think this module and subsequent classes should be primary_nav, it's more verbose but leaves far less room for ambiguity in the code.